docs/UserGuide.md

```diff
@@ -153,6 +157,11 @@ For valid `BUDGET_MONTH`, a positive number excluding any letters/symbols/spaces
 ---
 
 ### Delete Client: `delete -client`
+Deletes the specified client from the client list and subsequently deletes any pairings involving that client.
+
+<u>Format:</u> `delete -client ic/CLIENT_INDEX`
+
+<u>Example:</u> `delete -client ic/3`
```

Comment:
I really like how clear a explanation is.
Maybe can also consider having an expected result?